### DISCUSSION TITLE
URL parameter to SQL query

### DISCUSSION THREAD

**Discussion by mikkokotila** [grafana/grafana#38186] [Dashboards, Panels & Visualizations]
Basic case that most users will want to do at some point: 

- One dashboard generates a link like so `https://monitor.cavai.com/d/Q5QDvIJMk/test?editPanel=2&orgId=1&var-creative_id=4884`
- Another dashboard uses `creative_id` value `4884` in SQL query

At the moment the situation in the Grafana eco-system is that:

- a lot of people seem to be spending a lot of time to try to do this
- there are not canonical/comprehensive instructions/responses on how to do it
- there are no doc entries that show examples on how to do it

**Comment by mikkokotila** [comment]
Out of the several dozens of options, the one that actually does the job is this: https://stackoverflow.com/a/65003910/6941244

This project has a lot of potentials, but to have longevity for the long-term, developer/user experience must become a focus. Right now the learning curve hits very hard.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
